backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -32,9 +34,27 @@
 public class Translators {
   private static final DateTimeFormatter US_SLASHDATE_SHORT_FORMATTER =
       DateTimeFormatter.ofPattern("M/d/yyyy");
+
+  private static final long LOOK_BACK_YEARS = 99;
+
+  // Accepts either two-digit or four-digit years.
+  // Two-digit years will always resolve to years past.
+  // (e.g., if today is Jan. 1, 2023, and the formatter is passed 11/4/24, it will resolve to Nov 4,
+  // 1924).
+  public static final DateTimeFormatter PAST_DATE_FLEXIBLE_FORMATTER =
+      new DateTimeFormatterBuilder()
+          .appendPattern("M/d/")
+          .optionalStart()
+          .appendPattern("uuuu")
+          .optionalEnd()
+          .optionalStart()
+          .appendValueReduced(ChronoField.YEAR, 2, 2, LocalDate.now().minusYears(LOOK_BACK_YEARS))
+          .optionalEnd()
+          .toFormatter();
+
   private static final int MAX_STRING_LENGTH = 500;
 
-  public static final LocalDate parseUserShortDate(String d) {
+  public static final LocalDate parseUserDate(String d) {
```

Comment:
actually this isn't being used anywhere anymore, besides the tests

Reply:
I didn't realize this! Apparently all my concern over changing the `SHORT_FORMATTER` was unfounded 😆 Removed this method (and the old formatter), thanks for the tip.